locator/token_metadata.cc

```diff
@@ -176,6 +178,8 @@ class token_metadata_impl final {
 
     void add_leaving_endpoint(inet_address endpoint);
     void del_leaving_endpoint(inet_address endpoint);
+
+    void request_read_new();
```

Comment:
in the follow-up https://github.com/scylladb/scylladb/pull/13715 we are going to move `pending_ranges` to erm, and we would need this a flag in token_metadata, since the calculation will happen later, in `replicate_to_all_cores/create_effective_replication_map`

Reply:
But it still can be set in `update_pending_ranges` not as a separate call. Why I do not like the separate call is that, at least now, you need to know to call it before you call `update_pending_ranges` . This is not obvious and not clear.

Reply:
ok